v3/cypress/e2e/plugin.spec.ts

```diff
@@ -188,8 +188,25 @@ context("codap plugins", () => {
     // Edit formula
     table.editFormula(newName, "Mass * 2")
     webView.confirmAPITesterResponseContains(/"operation":\s"updateAttributes/)
+    // Edit formula also broadcasts an updateCases notification
+    webView.confirmAPITesterResponseContains(/"operation":\s"updateCases/)
     webView.clearAPITesterResponses()
 
+    cy.log("Broadcast updateCases notifications")
+    table.getGridCell(2, 2).dblclick()
+    table.getGridCell(2, 2).find("input").type("test{enter}")
+    webView.confirmAPITesterResponseContains(/"operation":\s"updateCases/)
+    webView.clearAPITesterResponses()
+
+    cy.log("Broadcast updateCollection notifications")
+    table.renameCollection("c1", "Mammals")
+    webView.confirmAPITesterResponseContains(/"operation":\s"updateCollection/)
+    webView.clearAPITesterResponses()
+
+    // TODO Check for deleteCollection notifications when deleting the last attribute
+    // in a grouped or ungrouped collection. I couldn't figure out how to do this because
+    // attribute dropdown menus don't work with mouseSensor but dragging won't work without it
```

Comment:
It took some doing, but I was able to figure out a test for deleting the last attribute in a grouped collection. I did it by opening a file that already contains a dataset with multiple collections (Four Seals) and deleting all the attributes in the grouped collection.

However, in trying to get this to work for the upgrouped collection, I discovered that deleting the last attribute in the ungrouped collection doesn't actually remove the ungrouped collection... it continues to exist as an empty collection. That's because `dataSet.removeAttribute` lacks a clause like `dataSet.setCollectionForAttribute` has, where `collection` is handled when `isCollectionModel(collection)` is false. Given that this is all going to get refactored soon, I'm not going to fix it in this PR. I'll create a PT bug story so we don't forget about it, though.